tests/model_registry/rest_api/test_model_registry_rest_api.py

```diff
@@ -59,47 +74,146 @@ def test_validate_model_registry_resource(
+                {
+                    "resource_name": "model_artifact",
+                    "api_name": "model_artifacts",
+                    "data": MODEL_FORMAT_NAME,
+                },
+                MODEL_FORMAT_NAME,
                 id="test_validate_updated_artifact_model_format_name",
             ),
             pytest.param(
-                {"modelFormatVersion": "v2"},
-                {"modelFormatVersion": "v2"},
+                {
+                    "resource_name": "model_artifact",
+                    "api_name": "model_artifacts",
+                    "data": MODEL_FORMAT_VERSION,
+                },
+                MODEL_FORMAT_VERSION,
                 id="test_validate_updated_artifact_model_format_version",
             ),
         ],
-        indirect=["updated_model_artifact"],
+        indirect=["updated_model_registry_resource"],
     )
     def test_create_update_model_artifact(
         self,
-        updated_model_artifact: dict[str, Any],
+        updated_model_registry_resource: dict[str, Any],
+        expected_param: dict[str, Any],
+    ):
+        """
+        Update model artifacts and ensure the updated values are reflected on the artifact
+        """
+
+        validate_resource_attributes(
+            expected_params=expected_param,
+            actual_resource_data=updated_model_registry_resource,
+            resource_name="model artifact",
+        )
+
+    @pytest.mark.parametrize(
+        "updated_model_registry_resource, expected_param",
+        [
+            pytest.param(
+                {
+                    "resource_name": "model_version",
+                    "api_name": "model_versions",
+                    "data": MODEL_VERSION_DESCRIPTION,
+                },
+                MODEL_VERSION_DESCRIPTION,
+                id="test_validate_updated_version_description",
+            ),
+            pytest.param(
+                {"resource_name": "model_version", "api_name": "model_versions", "data": STATE_ARCHIVED},
+                STATE_ARCHIVED,
+                id="test_validate_updated_version_state_archived",
+            ),
+            pytest.param(
+                {"resource_name": "model_version", "api_name": "model_versions", "data": STATE_LIVE},
+                STATE_LIVE,
+                id="test_validate_updated_version_state_unarchived",
+            ),
+            pytest.param(
+                {"resource_name": "model_version", "api_name": "model_versions", "data": CUSTOM_PROPERTY},
+                CUSTOM_PROPERTY,
+                id="test_validate_updated_version_custom_properties",
+            ),
+        ],
+        indirect=["updated_model_registry_resource"],
+    )
+    def test_updated_model_version(
+        self,
+        updated_model_registry_resource: dict[str, Any],
+        expected_param: dict[str, Any],
+    ):
+        """
+        Update, [RHOAIENG-24371] archive, unarchive model versions and ensure the updated values
```

Comment:
should we agree on a common way to add the ticket id? I noticed that between the 3 of use we use 3 different ways 🤣 cc @lugi0 . Not a blocker

Reply:
I don't know if it matters that much 😂 as long as we can ctrl+f for the Jira ID imho it's fine